v20.2/multi-region-setup.md

```diff
@@ -10,13 +10,13 @@ This page walks you through setting up a virtual environment for developing and
 
 1. Complete the previous section of the tutorial, [Create a Multi-Region Database Schema](multi-region-database.html).
 
-1. Make sure that you have the following installed on your local machine:
+2. Make sure that you have the following installed on your local machine:
 
-    - [CockroachDB](install-cockroachdb-mac.html)
-    - [Python 3](https://www.python.org/downloads/)
-    - [Pipenv](https://pipenv.readthedocs.io/en/latest/)
+- [CockroachDB](install-cockroachdb-mac.html)
```

Comment:
Hi Fabio!  This is still rendering weirdly in [the preview page](https://deploy-preview-9324--cockroachdb-docs.netlify.app/docs/v20.2/multi-region-setup.html).  To get this sublist to nest correctly, try indenting these bulleted links in by 4 spaces.